docs/_tutorials/phpqa-tutorial.md

```diff
@@ -59,14 +70,17 @@ If you want to see the help you can just use:
 phpqa generate -h
 ````
 
-If you want to create a *basic* PHPT test for the [`ucfirst` function][php-function-ucfirst]:
+If you want to create a *basic* PHPT test for [the `ucfirst` function][php-function-ucfirst]:
 
 ````bash
 phpqa generate -f ucfirst -b
 ````
 
-This will create a template PHPT file for the function ucfirst inside of your PHPT default folder. If you want to choose
-the PHPT directory where the files are generated you can pass if as the first argument of the generate command:
+This will create a template PHPT file for the function ucfirst inside of
+the `phpt/` directory.
+
+If you want to override the destination directory where the files are generated
+you can pass it as the first argument of the generate command:
```

Comment:
The recommended way, for now, is to always pass the target directory. Maybe this tutorial is not clear enough about it or even we could improve a bit the API in order to consider the directory as the current one when the `generate` command doesn't receive one.